ironfish/src/blockchain/blockchain.test.ts

```diff
@@ -103,6 +103,8 @@ describe('Blockchain', () => {
     expect((await chain.getHashAtSequence(2))?.equals(headerA1.hash)).toBe(true)
     expect((await chain.getHashAtSequence(3))?.equals(headerB2.hash)).toBe(true)
     expect((await chain.getHashAtSequence(4))?.equals(headerB3.hash)).toBe(true)
+
+    expect(await chain.db.size()).toBeGreaterThanOrEqual(size)
```

Comment:
I don't think this is testing much if we're accepting it can be equal as well. They could both be 0. I think removing this would be fine or just check strictly greater than. Ideally maybe it's own test too

Reply:
Updated to check the chain db size > 0. The size function is the approximate size https://github.com/Level/leveldown#dbapproximatesizestart-end-callback There is disclaimer that the result might not include the recent writes. In the previous test, the size value actually does not change due to this.